tutoraspects/templates/aspects/apps/superset/pythonpath/create_row_level_security.py

```diff
@@ -11,22 +11,27 @@
 
 session = security_manager.get_session()
 
-# Fetch the Open edX role
-role_name = "{{SUPERSET_OPENEDX_ROLE_NAME}}"
-openedx_role = session.query(Role).filter(Role.name == role_name).first()
-assert openedx_role, "{{SUPERSET_OPENEDX_ROLE_NAME}} role doesn't exist yet?"
-
-for (schema, table_name, group_key, clause, filter_type) in (
+row_level_security_filters = [
     (
         "{{ASPECTS_XAPI_DATABASE}}",
         "{{ASPECTS_XAPI_TABLE}}",
```

Comment:
I see we actually do have a new datasources for `transcript_usage` and `video_plays` now.. don't we need RLSFs for them too?

And so maybe the details for these filters should all live in the tutor config, so here we can iterate over `SUPERSET_ROW_LEVEL_SECURITY_FILTERS` + `SUPERSET_EXTRA_ROW_LEVEL_SECURITY_FILTERS`?